src/index.html

```diff
@@ -1,14 +1,40 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Enclosures</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Enclosures</h1>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Enclosures</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="first container" id="qa-block">
+      <div class="second">
+        <div class="third">
+        In programming languages, a closure is a technique for implementing lexically
+scoped name binding in a language with first-class functions. Operationally, a
+closure is a record storing a function together with an environment.
+        </div>
```

Comment:
```suggestion
        <div class="third">
          In programming languages, a closure is a technique for implementing lexically
          scoped name binding in a language with first-class functions. Operationally, a
          closure is a record storing a function together with an environment.
        </div>
```
check your formating